### ISSUE TITLE
Female Focused Symptoms/ Conditions Updating 2

### ISSUE THREAD

**Issue by Lucy-WhiteSwan** [obophenotype/human-phenotype-ontology#7512]
**Preferred term label:**
Pelvic Pain

**Synonyms**
Pain in the pelvis, chronic pelvic pain

**Definition (free text, please give PubMed ID)**
PubMed ID 32119472, Pain perceived in the area of the pelvis, the lower part of the abdomen located between the hip bones.

**Parent term (use hpo.jax.org/app)**
Pain

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
N/A

**Your nano-attribution (ORCID)**
N/A

**Comment by pnrobinson** [comment]
added as HP:0034267
All six requested terms will be available with the next HPO release in June 2022.

**Comment by Lucy-WhiteSwan** [comment]
Hi Peter, Ok, thank you.
